### ISSUE TITLE
Entmax-alpha in addons

### ISSUE THREAD

**Issue by rwbfd** [tensorflow/tensorflow#45905]
I would like to implement the entmax-alpha feature as is described in https://arxiv.org/abs/1905.05702 in the addons. For alpha=1, it is softmax, and for alpha=2, it is sparsemax. 

This feature is a key to realize sparse seq-to-seq model. It will not change the current API. 

I have almost finished implementing the feature. Therefore, I would like to know if this has any chance of being merged before continuing.

**Comment by amahendrakar** [comment]
@rwbfd,
Topics related to TensorFlow Addons are tracked in the tensorflow/addons repo. Could you please submit a new feature request from [this link](https://github.com/tensorflow/addons/issues/new/choose) so that it can be tracked there. Thanks!

**Comment by rwbfd** [comment]
@amahendrakar Thank you. I will close the issue now.
